### ISSUE TITLE
Tensorflow tutorial broken with "unexpected keyword argument 'half_pixel_centers'"

### ISSUE THREAD

**Issue by ic** [apache/incubator-tvm#4101]
Running the tutorial after compiling from source breaks with either TF1.14 or TF2 (I did not try any other version):

```
python tutorials/frontend/from_tensorflow.py
...
Traceback (most recent call last):

  File "tutorials/frontend/from_tensorflow.py", line 129, in <module>
    shape=shape_dict)

  File "tvm/python/tvm/relay/frontend/tensorflow.py", line 2375, in from_tensorflow
    mod, params = g.from_tensorflow(graph, layout, shape, outputs)

  File "tvm/python/tvm/relay/frontend/tensorflow.py", line 2020, in from_tensorflow
    op = self._convert_operator(node.op, inputs, attr, graph)

  File "tvm/python/tvm/relay/frontend/tensorflow.py", line 2338, in _convert_operator
    sym = convert_map[op_name](inputs, attrs, self._params)

  File "tvm/python/tvm/relay/frontend/tensorflow.py", line 411, in _impl
    extras={'method': "bilinear"})(inputs, attr)

  File "tvm/python/tvm/relay/frontend/common.py", line 412, in __call__
    return get_relay_op(op_name)(*inputs, **new_attrs)

TypeError: resize() got an unexpected keyword argument 'half_pixel_centers'
```

[Here is a WIP attempt](https://github.com/dmlc/tvm/compare/master...ic:6ec2b28) at introducing the attribute. Before committing more time, I would appreciate feedback/confirmation, as a definite new comer.

**Comment by DamonDeng** [comment]
Downgrading TF to 1.13.1 can resolve the issue. In case anyone else encounter this problem too.

The attribute "half_pixel_centers" was added to TF in tensorflow/tensorflow@3ae2c66 , which the pb file used in the TVM tutorial is not aware of.

Agree with @ic that we should modify the tutorial to support TF 1.14 and TF 2.0

**Comment by ic** [comment]
By the way, the tutorial is fixed in the linked PR (#4104).

**Comment by OriAlpha** [comment]
you can just go into the tensorflow graph and pop ou the attributes which create the errors. for example half_pixel_centers and then it works fine, I have faced these errors too
